test/lib/konnectors.spec.js

```diff
@@ -114,6 +82,18 @@ describe('konnectors lib', () => {
     })
   })
 
+  describe('isKonnectorUpdateNeededError', () => {
+    ;['TERMS_VERSION_MISMATCH'].forEach(errorType => {
+      it(`returns true for ${errorType}`, () => {
+        expect(
+          konnectors.isKonnectorUpdateNeededError(
+            konnectors.buildKonnectorError(errorType)
+          )
+        ).toBe(true)
+      })
+    })
+  })
+
```

Comment:
I see no tests on the propagateError mechanism. They seem to be important here, WDYT ?

Reply:
This mechanism is inside the big `AccountConnection` handmade store connected component, I don't a see a simple way to test it right now without refactoring all this legacy part :/